### ISSUE TITLE
Feature Request: Set rowHeight based on screen size

### ISSUE THREAD

**Issue by Jwink3101** [miromannino/Justified-Gallery#211]
I am not very experiences with any of the technologies used here, but I have been able to get a bare-bones gallery working (to be used on a personal static site builder).

Anyway, what I am struggling with is I want the images to be larger on a desktop and smaller on mobile. It would be great if I could specify conditional row heights.

As I said, I am not very expert, but the best I could wrangle for now is something like:

    if ( $(window).width() > 739) {      
        $("#gal").justifiedGallery({
            rowHeight : 200,
            maxRowHeight: 300,
            lastRow : 'nojustify',
            margins : 1
            });
    } 
    else {
        $("#gal").justifiedGallery({
            rowHeight : 80,
            maxRowHeight: 160,
            lastRow : 'nojustify',
            margins : 1
            });
    }

which works but requires a refresh in order for it to take affect. I will go with something like this unless someone here proposes something better, or the developer wishes to add it.

Thanks! This project really is awesome. And very well documented!

**Comment by miromannino** [comment]
Yes, I am working on this, definitely it's my first priority know.

**Comment by tomkeysers** [comment]
[This issue comment](https://github.com/miromannino/Justified-Gallery/issues/92#issuecomment-250480791) will give you a better temporary workaround.

**Comment by dassmilytier** [comment]
I was searching for the exact same thing. As this is still not closed yet, maybe other might be interested in an solution without breakpoints. This is what I cam up with:
`rowHeight: Math.max(80, Math.floor($(window).width() / 10)),`
where 80 is the smallest rowHight, while 10 is just a number you want to play around with.

**Comment by acwolff** [comment]
Suppose the height should be 20% of the screenheight, you could use this code (relThumbHeight=20)

        screenHeight = screen.height;
        if (screen.width < screen.height) screenHeight = screen.width;

            if (relThumbHeight > 0) {
                rowHeight_jg = relThumbHeight * screenHeight / 100;
                if (rowHeight_jg < 50) rowHeight_jg = 50;
                if (rowHeight_jg > maxThumbHeight) rowHeight_jg = maxThumbHeight;
            }
            $("#thumbsgallery").justifiedGallery({
                rowHeight: rowHeight_jg,
                maxRowHeight: maxRowHeight_jg,
            });
